vm/systemSmartContracts/esdt.go

```diff
@@ -228,7 +237,7 @@ func (e *esdt) checkBasicCreateArguments(args *vmcommon.ContractCallInput) vmcom
 		e.eei.AddReturnMessage(err.Error())
 		return vmcommon.UserError
 	}
-	if len(args.Arguments) < 2 {
+	if len(args.Arguments) < 1 {
```

Comment:
This change in checkBasicCreateArguments method from 2 to 1 does not create some backwards compatibility issues

Reply:
no. it does not create problems.